src/layout.ts

```diff
@@ -664,13 +725,33 @@ export const calculateFinalTokens = (
 
           sizer.scale.set(scaleWidth, scaleHeight);
 
-          fontProperties = { ...getFontPropertiesOfText(sizer, true) };
+          fontProperties = {
+            ...PIXI.TextMetrics.measureFont(
+              (sizer.style as PIXI.TextStyle).toFontString()
+            ),
+          };
 
           fontProperties.ascent *= scaleHeight;
           fontProperties.descent *= scaleHeight;
           fontProperties.fontSize *= scaleHeight;
 
-          const bounds = rectFromContainer(sizer);
+          const boundsCacheKey = JSON.stringify({
+            text: sizer.text,
+            scaleX: sizer.scale.x,
+            scaleY: sizer.scale.y,
+            style: stringifyStyle(sizer.style as PIXI.TextStyle),
```

Comment:
Why include style here? If you're looking to get only a PIXI.Rectangle then why not use the rectangle (serialized) as a key? There shouldn't be any reason you can't reuse them between objects with different text, styles, etc. 
Or am I missing something?

Reply:
I think i spoke too soon here. I see that it's not about storing the rectangle as much as avoiding repeating the measurements of the spacer.